cirq-google/cirq_google/workflow/io.py

```diff
@@ -66,8 +66,12 @@ def load(self, *, base_data_dir: str = ".") -> 'cg.ExecutableGroupResult':
             ],
         )
 
+    @classmethod
+    def _json_namespace_(cls) -> str:
+        return 'cirq.google'
+
     def _json_dict_(self) -> Dict[str, Any]:
-        return dataclass_json_dict(self, namespace='cirq.google')
+        return dataclass_json_dict(self, namespace=cirq.json_namespace(type(self)))
```

Comment:
this seems rather complicated. Why not `namespace=self._json_namespace_())`?

Reply:
Calling `cirq.json_namespace` is more friendly to changing `_json_namespace_` in a backwards-compatible way, as it limits the number of places that need to change to the `_json_namespace_` invocation in the protocol method. (We encountered something similar for `_resolve_parameters_` in the past.)

This is primarily a concern outside of Cirq, but since these are the only examples in Cirq I figured I should use this format to encourage its use elsewhere.